task_DEV_4/task_DEV_4/Rectangle.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace task_DEV_4
+{
+    class Rectangle
+    {
+
+        public int height;
+        public int width;
+
+        public Rectangle()
+        {
+
+        }
+
+        public Rectangle(int height, int width)
+        {
+            this.height = height;
+            this.width = width;
+        }
+
+        public void SettingRectangleValues(int height, int width)
+        {
+            this.height = height;
+            this.width = width;
+        }
+
+        public int CalculatingThePerimeterOfARectangle()
+        {
+            return (height + width) * 2;
+        }
+
+        public int CalculatingTheAreaOfARectangle()
+        {
+            return height * width;
+        }
+
+        public bool CheckForASquare()
```

Comment:
This method can be rewritten in one line. How?

Reply:
Better to name it `IsSquare`.

Reply:
Usually, articles are not written in the name of methods.

Reply:
done